examples/ingress/ingress.values.yaml

```diff
@@ -1,26 +1,14 @@
-# Using coder with ingress in versions from 1.21 and newer
-# Coder's built-in ingress controller is no longer packaged
-# the coderd pod does not require a fanout so it is exposed
-# by creating a LoadBalancer service
 coderd:
-  serviceNext: true
   devurlsHost: '*.devurls.coderhost.com'
   serviceSpec:
+    # The Ingress will route traffic to the internal ClusterIP.
     type: ClusterIP
-    # The values.yaml file in the chart includes LoadBalancer
-    # specs which need to have the keys removed using null
-    # this does not work as a sub-chart
-    loadBalancerIP: null
```

Comment:
@jawnsy I see this is already merged but as long as these loadBalancerIP, externalTrafficPolicy, and loadBalancerSourceRanges values are in the `values.yaml` file https://github.com/cdr/enterprise-helm/blob/main/values.yaml#L19 , the user is required to nullify them when changing the type to ClusterIP or NodePort.  Is the coderd service spec going to be changed to remove these lines?

Reply:
these seem to just be ignored for ClusterIP services, did you see something different?

Reply:
Kubectl doesn't complain about it but I added it because users were seeing errors about "service type ClusterIP doesn't support loadBalancerIP" and Helm uses `null` to remove the key from the yaml output.